### PR TITLE
[ADD] pos_solidarity_rounding, pos_solidarity_rounding_portal

### PR DESCRIPTION
**Pull request by carmenbianca**
## Odoo task (if applicable)

https://gestion.coopiteasy.be/web#id=10405&model=project.task&view_type=form&menu_id=

## Checklist before approval

- [ ] Tests are present (or not needed).
- [ ] Credits/copyright have been changed correctly.
- [ ] Change log snippet is present.
- [ ] (If a new module) Moving this to OCA has been considered.

### CODE REVIEW

**Comment by victor-champonnois** [comment]
@carmenbianca Why not proposing this dev to OCA/pos ?

File: pos_solidarity_rounding_portal/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+{
+    "name": "Point of Sale Solidarity Rounding Portal",
+    "summary": """
+        Allow registered users to change their solidarity rounding configuration in the
+        account settings.""",
+    "version": "12.0.1.0.0",
+    "category": "Uncategorized",
```
Comment (remytms) [inline pos_solidarity_rounding_portal/__manifest__.py]:
```suggestion
    "category": "Portal",
```

File: pos_solidarity_rounding_portal/controllers/portal.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+from odoo.addons.portal.controllers.portal import CustomerPortal
+
+
+class SolidarityPortal(CustomerPortal):
+    def __init__(self, *args, **kwargs):
+        super().__init__()
+        if "OPTIONAL_BILLING_FIELDS" not in vars(self):
+            self.OPTIONAL_BILLING_FIELDS = CustomerPortal.OPTIONAL_BILLING_FIELDS.copy()
+        self.OPTIONAL_BILLING_FIELDS.extend(["enable_solidarity_rounding"])
```
Comment (remytms) [inline pos_solidarity_rounding_portal/controllers/portal.py:15]: Interesting.

**Review by remytms — APPROVED** [review]

**Comment by carmenbianca** [comment]
@victor-champonnois I think it's a little too barebones/unconfigurable to propose to the OCA.

**Pull request merged**